### DISCUSSION TITLE
How do I replace BigCommerce's "order messages" in swell commerce?

### DISCUSSION THREAD

**Discussion by waves2cure** [swellstores/community#355] [Help] [answered]
I am using a BigCommerce at the moment and I am an energy healing service provider.  I provide healing services which may be for 1 week to 1 month at max. I have to communicate to my buyer as what kind of healing I performed on that particular day such as if energy transmission was done for auras, chakras, etheric cords, or whatever may be.  

In my present scenario in BigCommerce, I change order status as "Awaiting Shipment"  and then I push my messages using "order messages".  When all healing has ben completed, I mark my order as "Shipped".  I am trying to satisfy my business need with swell commerce platform.  Here, there are no order messages, then how do I communicate my activities of healing transmission with my customers?

I tried to speak with customer support through chat.  Initially they thought that I could customize order notifications, but that would also not fulfill the business requirement because I am not changing my order status every day.

I thought I need either "order messages" or something on a similar line of "shipment tracking" for physical products.  In this case, different line items of an order could be different shipment and for each shipment of the line item I should be able to add tracking details which could be pushed to my customers through emails.

Any guidance as how could I achieve my business requirements with swell commerce platform?

**Comment by logeshswell** [comment]
@waves2cure You can achieve your use case by using the custom notification feature that is available in Swell, following this guide:
https://community.swell.is/discussion/comment/1454/#Comment_1454

Basically, you have to create a custom notification titled 'Service update'.
Then, you can trigger this notification either manually via **Developer console** (or) through `conditions` field supported by custom notifications model (eg: when the note is added in the order events, trigger an email notification to the customer)

**Reply by logeshswell** [reply, marked as answer]
> I thought I need something on a similar line of "shipment tracking" for physical products. In this case, different line items of an order could be different shipment and for each shipment of the line item I should be able to add tracking details which could be pushed to my customers through emails.

**Alternate idea based on your comment:**

Yes, we do support separate shipments for each line item and each shipment will trigger an email notification with your tracking number (a.k.a details of the service provided). You can also customize the UI of the email template as per your needs.
